contracts/adapters/AaveV3LeverageStrategyExtension.sol

```diff
@@ -42,9 +42,11 @@ contract AaveV3LeverageStrategyExtension is AaveLeverageStrategyExtension {
     IPoolAddressesProvider public lendingPoolAddressesProvider;
     uint256 public maxOraclePriceAge;
     bool public overrideNoRebalanceInProgress;
+    IAaveOracle public aaveOracle;
 
     constructor(
         IBaseManager _manager,
+        IAaveOracle _aaveOracle,
```

Comment:
The aaveOracle should come from calling `getPriceOracle` on the `LendingPoolAddressProvider`

Reply:
Good point, thanks. Will adjust 👍

Reply:
Adjusted in:
https://github.com/IndexCoop/index-coop-smart-contracts/pull/142/commits/5d1cc5945ae49511d3ee546a0cccadad47fbd111

Reply:
In fact I went one step further and just read the address anew from the addressProvider everytime when I call the oracle.
Thereby we don't get rugged if for some reason the AaveOracle address changes:

https://github.com/IndexCoop/index-coop-smart-contracts/pull/142/commits/0a69def8d226b8921e98ad88c3158cb70ead0df6